### ISSUE TITLE
Broken with SublimeText 4 ?

### ISSUE THREAD

**Issue by tfrancart** [blake-regalia/linked-data.syntaxes#25]
Ubuntu recently updated my SublimeText package and LinkedData package is not working anymore.
SublimeText build = 4107

Anytthing special I need to do ?
Thanks !

**Comment by tfrancart** [comment]
I think it is the macaron-dark theme that is broken

Here is what I see

![image](https://user-images.githubusercontent.com/2728945/120607793-cff06c80-c450-11eb-8401-7cfe3c97f653.png)

**Comment by ncarboni** [comment]
same here, back in ST3 works perfectly

**Comment by paulwalk** [comment]
I am seeing this problem too. Selecting Preferences / "Color Scheme" / "auto" and then "mariana" seem to be a workaround, but only for the current file. I guess the problem is with the colour scheme.

**Comment by griffin-rickle** [comment]
Hey all,

TL;DR the LinkedData package is setting the color_scheme and overriding the global color scheme.

One workaround is going to `Settings -> Syntax Specific` and setting your color scheme there. However, as rwols pointed out in the ST thread, this package should ideally not be setting a Color Scheme at all. Here's the link:
https://forum.sublimetext.com/t/syntax-highlighting-broken-in-sublime-text-4/59175/15

**Comment by blake-regalia** [comment]
The colors being broken in ST4 is actually due to a bug in ST3 that was initially discovered when I opened an issue [here](https://github.com/facelessuser/ExportHtml/issues/67) and eventually led to [this bug report](https://github.com/sublimehq/sublime_text/issues/3176). Unfortunately it is going to require completely redoing the color schemes for it to look right in ST4. We can keep this issue open until such a fix is implemented for ST4.

As for overriding the syntax settings, this was done intentionally because this package creates super specific scopes and requires special color schemes to work properly, for example [see this comment](https://github.com/blake-regalia/linked-data.syntaxes/issues/3#issuecomment-518900170).

**Comment by paulwalk** [comment]
> One workaround is going to Settings -> Syntax Specific and setting your color scheme there.

Thanks @griffin-rickle - this is a good workaround.

**Comment by griffin-rickle** [comment]
@paulwalk The workaround works for now, but the whole point of the package is subverted; i.e. the granularity of highlighting subject, predicate, and object, datatype specifications, strings vs IRIs, etc with a custom color scheme is nowhere near as good as what you get with the color scheme this repo.

@blake-regalia 
> As for overriding the syntax settings, this was done intentionally because this package creates super specific scopes and requires special color schemes to work properly, for example [see this comment](https://github.com/blake-regalia/linked-data.syntaxes/issues/3#issuecomment-518900170).
You are absolutely correct. I attempted to take a look at the code and stop the forcing of a color scheme, but I see this is not the right way to go about it. I would like to help, but unfortunately I'm very new to the world of Sublime Text color scheme definitions and not a JavaScript coder by any means (that doesn't mean I'm not willing to learn). What's the first step to fixing the issue?

**Comment by blake-regalia** [comment]
Color schemes for ST4 fixed in new release!! v1.0.0 now available for install/update on PackageControl
@tfrancart @paulwalk @griffin-rickle @ncarboni

**Comment by tfrancart** [comment]
Alleluïa, updated ST4 Ubuntu package, and Turtle syntax highlight is back to normal. You made my day !

**Comment by griffin-rickle** [comment]
> Color schemes for ST4 fixed in new release!! v1.0.0 now available for install/update on PackageControl
> @tfrancart @paulwalk @griffin-rickle @ncarboni

Most excellent. Works perfectly again. Thank you!

**Comment by paulwalk** [comment]
confirmed - working perfectly now - thanks!

**Comment by ncarboni** [comment]
works great for me too! Thanks!
